sdk/spring/spring-cloud-azure-actuator/src/main/java/com/azure/spring/cloud/actuator/keyvault/KeyVaultCertificateHealthIndicator.java

```diff
@@ -34,12 +34,8 @@ protected void doHealthCheck(Health.Builder builder) {
             this.certificateAsyncClient.getCertificateWithResponse("spring-cloud-azure-not-existing-certificate")
                 .block(timeout);
             builder.up();
```

Comment:
This may never happen?

Reply:
may cover the case if it happens to have this key? we better keep it the original way.